docs/src/explanations/memories.md

```diff
@@ -200,3 +200,53 @@ Chisel memories can be initialized from an external `binary` or `hex` file emitt
 
 For more information, check the experimental docs on [Loading Memories](../appendix/experimental-features#loading-memories) feature.
 
+## SRAM
+
+Chisel provides an API to generate `SRAM` memories, which are currently alternative APIs for `SyncReadMem`. The key difference between the `SRAM` and `SyncReadMem` APIs is the former's capability to declare a specific number of read, write, and read-write memory ports, which are interacted with using explicit bundles.
```

Comment:
```suggestion
Chisel provides an API to generate `SRAM` memories, which are currently alternative APIs for `SyncReadMem`.
The key difference between the `SRAM` and `SyncReadMem` APIs is the former's capability to declare a specific number of read, write, and read-write memory ports, which are interacted with using explicit bundles.
```
As a general style when writing markdown, it's best to do 1 sentence per line--it makes diffs easier to read and merge conflicts easier to resolve.